conda_build/api.py

```diff
@@ -174,10 +174,11 @@ def skeletonize(packages, repo, output_dir=".", version=None, recursive=False,
     sources into expected conda recipe format."""
 
     version = getattr(config, "version", version)
+    #I dont now if these do anything
```

Comment:
@msarahan , do these do anything? version, outdir, and recursive are not in kwargs.